assets/app/actions/siteActions.js

```diff
@@ -55,13 +55,16 @@ export default {
     }).catch(error => alertActions.httpError(error.message));
   },
 
-  createCommit(site, path, fileData) {
+
+  createCommit(site, path, fileData, message = false, sha = false) {
     const b64EncodedFileContents = encodeB64(fileData);
-    const commit = {
-      message: `Adds ${path} to project`,
+    const siteId = site.id;
+    let commit = {
+      message: (message) ? message : `Adds ${path} to project`,
```

Comment:
Do we still need this fallback with the other default messages in place in the editor component?